dockerfiles/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM openjdk:8
+ARG HIVE_VERSION=3.0.0.13.3
+ARG HADOOP_VERSION_EE=3.2.0.12-EE-SNAPSHOT
+ARG PROMETHEUS_EXPORTER_VERSION="0.12.0"
+ARG USER="hive"
+ARG USERID="1515"
+ARG GROUP="hive"
```

Comment:
that is secure group id `metaserver`
https://github.com/logicalclocks/hops-hadoop-chef/blob/master/attributes/default.rb#L16